### PR TITLE
copy over joint_genotyping pipelines and GOTC docker to warp

### PR DESCRIPTION
**Pull request by nikellepetrillo**

### CODE REVIEW

**Comment by kbergin** [comment]
@jessicaway @gbggrant and I were chatting about the "by chromosome" JG pipeline and whether we wanted it in WARP without tests. The hope is that the new HAIL pipeline will replace it soon, but we discussed that maybe in the meantime the pipeline should live in the beta-pipelines/ folder instead of in the main pipelines folder. Thoughts on moving that around?

**Comment by jessicaway** [comment]
I think we can make a separate PR to move by chromosome into a beta pipelines folder

File: pipelines/broad/dna_seq/germline/joint_genotyping/by_chromosome/JointGenotypingByChromosomePartOne.wdl [hunk 1]
```diff
@@ -0,0 +1,269 @@
+version 1.0
+
+import "../../../../../../tasks/JointGenotypingTasks.wdl" as Tasks
```
Comment (sophiacrennan) [inline pipelines/broad/dna_seq/germline/joint_genotyping/by_chromosome/JointGenotypingByChromosomePartOne.wdl:3]:
```suggestion
import "../../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks
```

File: pipelines/broad/dna_seq/germline/joint_genotyping/by_chromosome/JointGenotypingByChromosomePartTwo.wdl [hunk 2]
```diff
@@ -0,0 +1,228 @@
+version 1.0
+
+import "../../../../../../tasks/JointGenotypingTasks.wdl" as Tasks
```
Comment (sophiacrennan) [inline pipelines/broad/dna_seq/germline/joint_genotyping/by_chromosome/JointGenotypingByChromosomePartTwo.wdl:3]:
```suggestion
import "../../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks
```

File: pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl [hunk 3]
```diff
@@ -1,6 +1,6 @@
 version 1.0
 
-import "../../../../tasks/JointGenotypingTasks.wdl" as Tasks
+import "../../../../../tasks/JointGenotypingTasks.wdl" as Tasks
```
Comment (sophiacrennan) [inline pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl:3]:
```suggestion
import "../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks
```

**Comment by nikellepetrillo** [comment]
passing tests ran on master here: 
Exome: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/165/console
WGS: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/170/console

**Review by jessicaway — APPROVED** [review]

**Pull request merged**